lib/git-process.ts

```diff
@@ -224,7 +224,7 @@ export class GitProcess {
         }
       })
 
-      if (options && options.stdin) {
+      if (options && options.stdin !== undefined) {
```

Comment:
Do we care about `null` here?

Reply:
@iAmWillShepherd I personally don't think we should since the options type defines the stdin property as an optional string. If people run typescript without strict null checks they're in the danger zone already. Like, it's the same thing as how nodejs' `Writeable.write` is typed as `string | buffer`. You're not expected to pass null there but it's yabbaland so you can obviously do whatever you want.

That said I'm fine with whatever. I've worked around this bug already in desktop so there's no rush. Just some sad code that we can get rid of in desktop once this lands.

Reply:
Fair enough. I noticed the strict equality check and was just curios if `null` was ok.